src/styles/blocks/calendar.scss

```
@@ -0,0 +1,37 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+@import "../utils/functions";
+@import "../normalize/normalize.scss";
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cell-gap;
+  width: 7 * $cell-size + 10 * $cell-gap;
+
+  &__cell {
+    @include square ($cell-size);
+    background-color: #{$cell-color};
+    border: #{$cell-border};
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: all 0.5s ease;
+    cursor: pointer;
+
+    &:hover {
+      background-color: palevioletred;
+      transform: translate(0, -20px);
+    }
+
+    &::before {
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
+      color: black;
```

don't forget to create variables for colors

```suggestion
      color: $c-black;
```